src/novas.c

```diff
@@ -697,7 +697,7 @@ int set_planet_provider_hp(novas_planet_provider_hp func) {
 int place_star(double jd_tt, const cat_entry *star, const observer *obs, double ut1_to_tt, enum novas_reference_system system,
         enum novas_accuracy accuracy, sky_pos *pos) {
   static const char *fn = "place_star";
-  object source = { };
+  object source = {0};
```

Comment:
You are right about these, and in fact I had them in before, but then I noticed gcc does not seem to care (at least with `-Wall`) even as far back as gcc 2.90, so I removed the zeroes to stay minimalistic. But, I agree, that putting them back in is more standard... Thanks.

Reply:
Yeah, the story for a lot of this stuff is gcc (and GNU C) is perfectly happy with non-standard compliant code :)